settings.gradle.kts

```diff
@@ -34,3 +34,4 @@ include(":prices")
 include(":menus")
 include(":snackbars")
 include(":badges")
+include(":quantities")
```

Comment:
I suggest a name change here.
Something along the lines of `quantity-pickers` since quantities is not a category of component.
I'm just opening the discussion.